clang/lib/CIR/CodeGen/CIRGenExpr.cpp

```diff
@@ -216,13 +216,16 @@ static bool isAAPCS(const TargetInfo &TargetInfo) {
   return TargetInfo.getABI().startswith("aapcs");
 }
 
-Address CIRGenFunction::getAddrOfField(LValue base, const FieldDecl *field,
-                                       unsigned index) {
+Address CIRGenFunction::getAddrOfBitFieldStorage(LValue base, 
+                                                 const FieldDecl *field,
+                                                 unsigned index, 
+                                                 unsigned size) { 
   if (index == 0)
     return base.getAddress();
 
   auto loc = getLoc(field->getLocation());
-  auto fieldType = convertType(field->getType());
+  auto fieldType = builder.getUIntNTy(size);
```

Comment:
Please explain why it shouldn't use `convertType` here.

Reply:
`convertType` for the integer field is always 32,  type mismatch guaranteed

Reply:
So please add `assert(convertType(field->getType()) == ...)`